annotations.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	mkr "github.com/mackerelio/mackerel-client-go"
+	"github.com/mackerelio/mkr/logger"
+	"gopkg.in/urfave/cli.v1"
+	"os"
+)
+
+var commandAnnotations = cli.Command{
```

Comment:
It's better to add "Description" field like monitors one.

```
	Description: `
    Manipulate monitor rules. With no subcommand specified, this will show all monitor rules.
    Requests APIs under "/api/v0/monitors". See https://mackerel.io/api-docs/entry/monitors .
`,
```